zerver/webhooks/gci/doc.md

```diff
@@ -0,0 +1,14 @@
+If you are a participating organization with
+[Google Code-in](https://developers.google.com/open-source/gci/),
+you can now get Task notifications in Zulip!
+
+{!create-stream.md!}
+
+{!create-bot-construct-url.md!}
+
+To set up the webhook, send the URL you constructed above to
+gci-support@google.com.
```

Comment:
I expanded the text to discuss the fact that this integration will be incomplete until we get more event types documented.